wrapper/Ada/user_settings.h

```diff
@@ -260,10 +260,10 @@ extern "C" {
 
 
 /* Openssl compatibility */
+#define OPENSSL_ALL
```

Comment:
Not a fan of having to enable and pull in 100% of the compatibility layer. Once https://github.com/wolfSSL/wolfssl/pull/8258 is merged just use OPENSSL_EXTRA.
Unless you can find a way to avoid it.

Reply:
I wasn't a fan of it either, but couldn't see another way without removing the guard on that function or completely re-implementing it in the Ada C wrapper. I'll update this with OPENSSL_EXTRA today.